### DISCUSSION TITLE
Where are the Article Preview images defined in a post? (web.brid.gy)

### DISCUSSION THREAD

**Discussion by TomCasavant** [snarfed/bridgy-fed#1216] [Q&A] [answered]
Maybe I missed it in the docs, but is it possible for me to customize which images are shown along with the article URL, or is it possible for me to remove them entirely?

![Screenshot from 2024-07-26 20-25-01](https://github.com/user-attachments/assets/d1682bae-fb04-4e07-b563-f06b53fa06aa)

Edit: Also, I guess while I'm here the first blog post I bridged I had incorrect formatting for the post-url which caused the post for https://tomcasavant.com/blogrolls-on-blogrolls-on-blogrolls/ to link to https://fed.brid.gy/r/https://tomcasavant.com/2-904-hours-later-a-pokemon-saga/, I tried sending webmentions for both of the pages to fix that post but nothing seems be updating it

**Comment by snarfed** [comment, marked as answer]
Hmm! If you publish via webmention, BF should only attached images like that if you have the h-photo microformats2 on them in your HTML: https://fed.brid.gy/docs#web-image , https://fed.brid.gy/docs#web-which-parts , https://fed.brid.gy/docs#web-how-post

Otherwise, if that post came from an RSS/Atom feed, BF attaches images that it finds in enclosures or from OGP or Twitter card metadata on the post. There was a brief period recently where BF attached your site's favicon(s) as images to posts, but it didn't last long, and it shouldn't be doing that now.

**Reply by TomCasavant** [reply]
Hm, yeah I guess at one point I must have had u-photo on the opengraph images and didn't realize it

Fetched the post from activitypub.academy and it handled it properly ![Screenshot_20240727_115738_Firefox.jpg](https://github.com/user-attachments/assets/2d9c3b16-e7d6-482f-b8ea-e55751a41626)

I guess mastodon didn't process the media update
